### ISSUE TITLE
CC Contacts don't get notified on AR Publication

### ISSUE THREAD

**Issue by ramonski** [bikalims/bika.lims#2280]
## Steps to reproduce

- Create a new Client
- Add 2 Contacts (A and B) to that Client including Email and set the Publication Preference to Email
- Create a new AR for this Client
- Add "Contact A" as the primary contact and "Contact B" as the CC Contact in the AR Add form
- Publish the AR 

## Current behavior

Only the primary contact is notified via email

## Expected behavior

CC Contacts also get notification emails if their Publication Preferences are set to Email

## Screenshot (optional)

<img width="946" alt="ramon_bartl_" src="https://user-images.githubusercontent.com/713193/32272347-fd1ae700-befd-11e7-9b89-16728c699ae7.png">

**Comment by ramonski** [comment]
Publication Preferences are fetched from the primary contact instead of the CC Contact:
https://github.com/bikalims/bika.lims/blob/master/bika/lims/browser/analysisrequest/publish.py#L1090

```python
    def get_recipients(self, ar):
        """Returns a list with the recipients and all its publication prefs
        """
        recips = []

        # Contact and CC's
        contact = ar.getContact()
        if contact:
            recips.append({'title': to_utf8(contact.Title()),
                           'email': contact.getEmailAddress(),
                           'pubpref': contact.getPublicationPreference()})
        for cc in ar.getCCContact():
            recips.append({'title': to_utf8(cc.Title()),
                           'email': cc.getEmailAddress(),
                           # XXX This should be cc.getPublicationPreference()
                           'pubpref': contact.getPublicationPreference()})

        return recips
```

**Comment by ramonski** [comment]
Fixed in 3.2.1, 3.3.0 and master
